test/e2e/src/pages/diagramDetail.ts

```diff
@@ -20,6 +20,8 @@ export class DiagramDetail {
   }
 
   public async clickOnElement(elementId: string): Promise<void> {
+    browser.wait(ExpectedConditions.visibilityOf(this._bpmnIoContainer), browser.params.defaultTimeoutMS);
```

Comment:
Based on what I've seen in other PRs, shouldn't this be:

```suggestion
    await browser.wait(ExpectedConditions.visibilityOf(this._bpmnIoContainer), browser.params.defaultTimeoutMS);
```

?